web/app/themes/justicejobs/webpack.mix.js

```diff
@@ -16,6 +16,7 @@ mix.js([
     .sass('src/scss/style.scss', dist + 'css/main.min.css')
     .sass('src/scss/errors/404.sass', dist + 'css/404.css')
     .sass('src/scss/errors/error-page.sass', dist + 'css/error-page.css')
+    .sass('src/scss/ie.scss', dist + 'css/ie.min.css')
```

Comment:
I would suggest we dont need a ie separate style sheet. The styles in ie.scss were for IE10+ so do not work with conditional tags. We currently do not have any specific styles for IE9 or below so I would remove this extra CSS file and the enqueue. Happy to discuss. 

I see you have added a new mixin for IE10&11 .  we could probably delete the ie sass file and move the styles as you have done elsewhere.